src/Modules/Build.lua

```diff
@@ -937,8 +937,10 @@ function buildMode:OnFrame(inputEvents)
 				end
 		elseif IsKeyDown("CTRL") then
 				if event.key == "i" then
-						self.viewMode = "IMPORT"
+					self.viewMode = "IMPORT"
 					self.importTab:SelectControl(self.importTab.controls.importCodeIn)
+				elseif event.key == "m" and self.viewMode == "TREE" then
+					self.treeTab:OpenSpecManagePopup()
```

Comment:
I'm not really sure this shortcut is needed...how often do people need to manage their trees?  In addition, tab-specific shortcuts should really go on those tabs instead of in Build.lua